Makefile

```diff
@@ -14,9 +19,9 @@ NODE_BIN := $(THIS_DIR)/node_modules/.bin
 
 # applications
 NODE ?= node
-NPM ?= $(NODE) $(shell which npm)
+NPM ?= npm
```

Comment:
I'm not sure there was a reason for it, but previously it made sure that npm was run using the same node as the rest of the process. You could supply a NODE=/my/fancy/node as an evironment variable and have that node used. 

I don't think we ever actually used that capability though?